Source/WebCore/platform/ThemeTypes.h

```diff
@@ -33,12 +33,12 @@ class TextStream;
 
 namespace WebCore {
 
-enum SelectionPart {
+enum class SelectionPart : bool {
     SelectionBackground,
     SelectionForeground
```

Comment:
Since they are now namespaced it would be better to rename these to Background and Foreground, no? Same below.